ql/src/java/org/apache/hadoop/hive/ql/io/AcidUtils.java

```diff
@@ -2265,6 +2268,55 @@ public static Boolean isToInsertOnlyTable(Table tbl, Map<String, String> props)
     return true;
   }
 
+  public static Boolean isToFullAcid(Table table, Map<String, String> props) {
+    String transactional = props.get(hive_metastoreConstants.TABLE_IS_TRANSACTIONAL);
```

Comment:
can we simplify this a bit?
````
if (table == null || !TxnUtils.isTransactionalTable(props)) {
   return false;
}
````
I don't understand how it's possible that transactionalProp!= null, but transactional = null.